### ISSUE TITLE
NTR: alpha-aminoadipic acid high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8491]
New term request alpha-aminoadipic acid high in blood (segal_030430182137): 2-aminoadipic

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000278
- label: Elevated circulating 2-aminoadipic acid concentration
- parents: [Abnormal circulating dicarboxylic acid concentration (HP:0010995)](https://hpo.jax.org/app/browse/term/HP:0010995)
- pmid: PMID:23141293
- def: The concentration of 2-aminoadipic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8491
- synonyms:
